zebra-chain/src/serialization/error.rs

```diff
@@ -50,6 +51,10 @@ pub enum SerializationError {
     /// rule](https://zips.z.cash/protocol/protocol.pdf#txnencodingandconsensus).
     #[error("transaction balance is non-zero but doesn't have Sapling shielded spends or outputs")]
     BadTransactionBalance,
+
+    /// Invalid coinbase transaction.
+    #[error("coinbase error: {0}")]
+    Coinbase(coinbase::Error),
```

Comment:
You can hopefully remove the `From` implementation below with this:

```suggestion
    Coinbase(#[from] coinbase::Error),
```